astropy/io/fits/column.py

```diff
@@ -130,6 +130,30 @@
 # converting from binary to ASCII tables
 DEFAULT_ASCII_TNULL = '---'
 
+# Column-specific time override keywords
+COLUMN_TIME_KEYWORDS = {'TCTYP' : 'scale',
+                        'TRPOS' : 'pos',
+                        'TCUNI' : 'unit'}
+
+
+def is_time_column_keyword(keyword):
+    """
+    Check if the FITS header keyword is a time column-specific keyword.
+
+    Parameters
+    ----------
+    keyword : str
+        FITS keyword.
+    """
+
+    try:
+        key, idx = re.match(r'([A-Z]+)([0-9]+)', keyword).groups()
```

Comment:
It's a little more clear in the code to use the idiom below instead of catching an attribute error:
```
match = re.match(r'...', ..)
if match:
    key, idx = match.groups()
    ...
else:
    ...
```
In this case you don't care about `idx` so it could be even simpler:
```
match = re.match(r'...', ..)
return (match and match.group(1) in COLUMN_TIME_KEYWORDS)
```